compendium/DeclarativeServices/src/manager/ComponentConfigurationImpl.cpp

```diff
@@ -255,4 +255,10 @@ InstanceContextPair ComponentConfigurationImpl::CreateAndActivateComponentInstan
   return std::make_pair(componentInstance, ctxt);
 }
 
+void ComponentConfigurationImpl::UnbindAndDeactivateComponentInstanceHelper(std::shared_ptr<ComponentInstance> componentInstance)
```

Comment:
how is this function called? I don't see any callers of this function.

Reply:
And then removed.